tools/hyperref-check.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# Extracting all broken hyperref detected by PDFTex
+for file in "./main/acle.md" "./morello/morello.md" "./mve_intrinsics/mve.md" "./neon_intrinsics/advsimd.md"; do
+  echo "Checking $file..."
+
```

Comment:
Maybe add a comment with an example of the string that we want to extract from the output of pandoc? So that it will be easier to understand what `grep` and `sed` are doing.